Project.ConsoleApp.Test/ConsoleAppTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+
+namespace Project.ConsoleApp.Test
+{
+    [TestClass]
+    public class ConsoleAppTest
+    {
+        string _validJSONFile = string.Empty;
+        string _invalidJSONFile = string.Empty;
+        string _validXMLFile = string.Empty;
+        string _invalidXMLFile = string.Empty;
+
+        public ConsoleAppTest()
+        {
+            string testFilePath = Directory.GetCurrentDirectory() + @"\TestFiles\";
+
+            _validJSONFile = testFilePath + "JSON_ValidFormat.json";
+            _invalidJSONFile = testFilePath + "JSON_InvalidFormat.json";
+            _validXMLFile = testFilePath + "XML_ValidFormat.xml";
+            _invalidXMLFile = testFilePath + "XML_InvalidFormat.xml";
+        }
+
+        [TestMethod]
+        public void AllTestFilesExist()
+        {
+            Assert.IsTrue(File.Exists(_validJSONFile));
+            Assert.IsTrue(File.Exists(_invalidJSONFile));
+            Assert.IsTrue(File.Exists(_validXMLFile));
+            Assert.IsTrue(File.Exists(_invalidXMLFile));
+        }
+
+        [TestMethod]
+        public void Arguments_WithValidArguments()
+        {
+            //assert
+            try
+            {
+                Program.Main(new string[] { "-json", _validJSONFile, "-xml", _validXMLFile });
+            }
+            catch (ArgumentException aex)
+            {
+                Assert.Fail(aex.Message);
+            }
+        }
+
+        [TestMethod]
+        public void Arguments_WithInvalidArguments()
+        {
+            //assert
+            try
+            {
+                Program.Main(new string[] { "json", _validJSONFile, "xml", _validXMLFile });
+            }
+            catch (ArgumentException aex)
+            {
+                Assert.IsTrue(aex.Message == "Invalid arguments.");
```

Comment:
`Observation`: This test is very brittle. If the program doesn't throw an exception, then it will pass (with invalid arguments).  Additionally, if someone drops the period in the message (i.e. "Invalid Arguments") then this test will fail when really it is still working fine.  Match strings in constants or optionally don't match at all since the test probably doesn't care what the actual message is.  Also, why didn't you use `Assert.ThrowsException()` here?